index.html

```diff
@@ -0,0 +1,209 @@
+    </header>
+    <header>
+        <h2 style="color: #ffbf1d; text-align: left">Habilidades</h2>
+        <ul style="list-style-type: none">
+            <li style="border: 2px solid whitesmoke; padding: 3px; width: 25%; color: crimson; text-align: left">Javascript</li>
+            <li style="border: 2px solid whitesmoke; padding: 3px; width: 25%; color: crimson; text-align: left">Html e Css</li>
+            <li style="border: 2px solid whitesmoke; padding: 3px; width: 25%; color: crimson; text-align: left">JQuery</li>
+            <li style="border: 2px solid whitesmoke; padding: 3px; width: 25%; color: crimson; text-align: left">Wordpress</li>
+        </ul>
+    </header>
+</section>
+<section style="text-align: center">
+    <header>
+        <h3 style="color: #ff0b68; padding-top: 3%; font-size: 250%">Contato</h3>
+        <p style="color: #ffbf1d">Quentin Jacobsen tem uma paixão platônica pela magnífica vizinha e colega de escola Margo Roth Spiegelman.</p>
+        <p style="color: #ffbf1d">sestelos@gmail.com</p>
+        <p style="color: #ffbf1d">+55 21 98899-3752</p>
+    </header>
+    <header>
+        <h2 style="text-align: center; color: #faffbd">Midias</h2>
+        <nav>
+            <ul style="display: inline-flex; list-style-type: none">
+                <li>
+                    <a href="">
+                        <img src="images/social-facebook-circular-button.svg" alt="facebook-icon" style="background-color: rgba(255,230,38,0.7); height: 30px">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="images/twitter-circular-button.svg" alt="facebook-icon" style="background-color: rgba(255,230,38,0.7); height: 30px">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="images/linkedin-logo.svg" alt="facebook-icon" style="background-color: rgba(255,230,38,0.7); height: 30px">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="images/pinterest-ring-yellow-icon.svg" alt="facebook-icon" style="background-color: rgba(255,230,38,0.7); height: 30px">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="images/dribbble-icon.svg" alt="facebook-icon" style="background-color: rgba(255,230,38,0.7); height: 30px">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="images/google+.svg" alt="facebook-icon" style="background-color: rgba(255,230,38,0.7); height: 30px">
+                    </a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+</section>
+<footer>
+    <p style="color: #faffbd; text-align: center">Women + Tech = < /strong ></p>
```

Comment:
Utilize caracteres especiais para garantir que esses caracteres não sejam confundidos com elementos funcionais do seu código. Referência: https://css-tricks.com/snippets/html/glyphs/

```suggestion
    <p style="color: #faffbd; text-align: center">Women + Tech = &lt; /strong &gt;</p>
```